orca-applications/src/main/groovy/com/netflix/spinnaker/orca/applications/pipelines/CreateApplicationStage.groovy

```diff
@@ -17,23 +17,17 @@
 package com.netflix.spinnaker.orca.applications.pipelines
 
 import com.netflix.spinnaker.orca.applications.tasks.UpsertApplicationTask
-import com.netflix.spinnaker.orca.pipeline.LinearStage
-import com.netflix.spinnaker.orca.pipeline.model.Stage
+import com.netflix.spinnaker.orca.pipeline.StageDefinitionBuilder
 import groovy.transform.CompileStatic
-import org.springframework.batch.core.Step
 import org.springframework.stereotype.Component
 
 @Component
 @CompileStatic
-class CreateApplicationStage extends LinearStage {
-  public static final String PIPELINE_CONFIG_TYPE = "createApplication"
-
-  CreateApplicationStage() {
-    super(PIPELINE_CONFIG_TYPE)
-  }
-
+class CreateApplicationStage implements StageDefinitionBuilder {
```

Comment:
`LinearStageDefinitionBuilder` is an adapter over `LinearStage` that can handle a `StageDefinitionBuilder`.

Will need to revise this strategy a bit when migrating over other types of `StageBuilder`.